kubernetes/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 ### Changes
 
 * [FEATURE] add an option to collect node labels as host tags
+* [IMPROVEMENT] add custom tags to service checks
```

Comment:
Can you please add the PR number? Already fixed the first line's missing number in another PR to be merged